app/src/main/res/layout/fragment_new_message.xml

```diff
@@ -612,17 +612,23 @@
                                     android:visibility="gone"
                                     app:icon="@drawable/ic_editor_clock" />
 
-                                <com.google.android.material.button.MaterialButton
+                                <FrameLayout
                                     android:id="@+id/editorAi"
-                                    style="@style/EditorIconButton"
                                     android:layout_width="@dimen/newMessageToolbarItemWidth"
                                     android:layout_height="match_parent"
-                                    android:contentDescription="@string/buttonSchedule"
-                                    android:theme="@style/AiCursorAndPrimaryColorTheme"
-                                    android:visibility="gone"
-                                    app:icon="@drawable/ic_ai_magic_wand"
-                                    app:iconTint="@color/aiColor"
-                                    tools:visibility="visible" />
+                                    android:background="?android:attr/selectableItemBackgroundBorderless">
+
+                                    <com.airbnb.lottie.LottieAnimationView
+                                        android:layout_width="@dimen/standardIconSize"
+                                        android:layout_height="@dimen/standardIconSize"
+                                        android:layout_marginHorizontal="16dp"
```

Comment:
Do you really need this margin? Also use the margin constants if you really need it

Reply:
What do you need this margin for because I see no point in using it